README.md

```diff
@@ -179,22 +179,6 @@ pip install -r requirements.txt
 
 After installing the required dependencies, you can run the transcoder with the following:
 ```
-python main.py --help
-```
-
-## Creating a shortcut
-
-If you prefer to make the transcoder available globally on your machine, execute the scripts below. This script will update the permissions on the txcode symlink to grant execute permissions, and add the txcode symlink to your system PATH variable.
-
-Note if using macOS with interactive shells, you will need to change '\~/.bash_profile' to '\~/.bash_rc'.
-
-```
-cd market-data-transcoder/
```

Comment:
@salsferrazza I think at this point we can remove bin/txcode? The PyPI setup is referencing the main.py file directly for the alias.

Reply:
I agree, as long as `txcode` is addressable after installation having it in `/bin` is superfluous.

Reply:
bin is removed, and updated the examples documentation to use 'txcode'